From b651ee916f854adea9a77d24c03803b912eaba11 Mon Sep 17 00:00:00 2001
From: Wang Dongsheng <dongsheng.wang@nxp.com>
Date: Fri, 21 Apr 2017 13:26:07 +0800
Subject: [PATCH 009/767] arm64: dts: ls1043a: add ftm0 nodes

Add rcpm and ftm0 nodes. The Power Management related features
need these nodes.

Signed-off-by: Zhang Ying-22455 <ying.zhang22455@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
index 56b4c648a0fc..7e456294a096 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1043a.dtsi
@@ -677,6 +677,16 @@
 			status = "disabled";
 		};
 
+		ftm0: ftm0@29d0000 {
+			compatible = "fsl,ftm-alarm";
+			reg = <0x0 0x29d0000 0x0 0x10000>,
+			      <0x0 0x1ee2140 0x0 0x4>;
+			reg-names = "ftm", "FlexTimer1";
+			interrupts = <0 86 0x4>;
+			big-endian;
+			status = "okay";
+		};
+
 		wdog0: wdog@2ad0000 {
 			compatible = "fsl,ls1043a-wdt", "fsl,imx21-wdt";
 			reg = <0x0 0x2ad0000 0x0 0x10000>;
-- 
2.17.0

